yarn.lock

```diff
@@ -3045,6 +3045,13 @@
   dependencies:
     "@testing-library/dom" "^7.16.2"
 
+"@testing-library/user-event@^12.6.0":
```

Comment:
(refactor/blocking) - can you try regenerating the yarn lock again? it's generating differently for me - is it possible you updated from the main branch after you last ran `yarn/npm install`?

Reply:
Yes I did update with the master branch before I moved the PR out of draft.

Reply:
Thanks for updating it - we recently bumped some storybook versions and it looks like something went a bit wrong (not in your commit!). Either way, I'm now not having any issues running the install with your branch 👍